src/pages/_app.tsx

```diff
@@ -19,7 +21,6 @@ import { VotingProvider } from 'store/contexts/votingContext'
 import { VeNewoProvider } from 'store/contexts/veNewoContext'
 
 import 'styles/date-range-picker.css'
-import 'styles/react-markdown.css'
```

Comment:
This is the styling for the view snapshot pages.

Reply:
fixed

Reply:
i'm still not seeing this import on the `_app.tsx` can you please double check? thanks